From 95c08c5b52ad36562cca279ba50de39f80e37ca6 Mon Sep 17 00:00:00 2001
From: Julien Pivotto <roidelapluie@inuits.eu>
Date: Tue, 22 Sep 2020 11:56:16 +0200
Subject: [PATCH] Eureka: Fix service discovery when compiled in 32-bit

Java timestamps are causing issues when unmarshalling with a 32 bit
prometheus. It appears that we do not use those fields, so let's remove
them.

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>
---
 discovery/eureka/client.go | 2 --
 1 file changed, 2 deletions(-)

diff --git a/discovery/eureka/client.go b/discovery/eureka/client.go
index 6d45fb77e6..5840f968c8 100644
--- a/discovery/eureka/client.go
+++ b/discovery/eureka/client.go
@@ -55,8 +55,6 @@ type Instance struct {
 	DataCenterInfo                *DataCenterInfo `xml:"dataCenterInfo"`
 	Metadata                      *MetaData       `xml:"metadata"`
 	IsCoordinatingDiscoveryServer bool            `xml:"isCoordinatingDiscoveryServer"`
-	LastUpdatedTimestamp          int             `xml:"lastUpdatedTimestamp"`
-	LastDirtyTimestamp            int             `xml:"lastDirtyTimestamp"`
 	ActionType                    string          `xml:"actionType"`
 	CountryID                     int             `xml:"countryId"`
 	InstanceID                    string          `xml:"instanceId"`
